ts/features/walletV3/payment/store/reducers/index.ts

```diff
@@ -0,0 +1,174 @@
+import * as pot from "@pagopa/ts-commons/lib/pot";
```

Comment:
could it be a good idea to divide this reducer into multiple files?

Reply:
In that case you need to split the reducer in multiple reducers. I don't think it's a good idea.